Add BCD for CSS scale, rotate, and translate properties

chrisdavidmills opened the pull request:
As per https://bugzilla.mozilla.org/show_bug.cgi?id=1207734

teoli2003 commented on css/properties/rotate.json:
```diff
@@ -0,0 +1,116 @@
+        },
+        "x_y_z": {
+          "__compat": {
+            "description": "Support for x, y, and z axis values",
```
It is not clear what this relate too on the MDN page.

What about "Three length/percentage values" ? Shorter and match the text on the page.

chrisdavidmills commented:
I've looked at this, updated the wording on the page slightly (see https://developer.mozilla.org/en-US/docs/Web/CSS/rotate#Values), and updated this wording to match it.

teoli2003 commented on css/properties/rotate.json:
```diff
@@ -0,0 +1,116 @@
+{
+  "css": {
+    "properties": {
+      "translate": {
```
"rotate"

teoli2003 commented on css/properties/rotate.json:
```diff
@@ -0,0 +1,116 @@
+    "properties": {
+      "translate": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/translate",
```
"rotate"

teoli2003 commented on css/properties/rotate.json:
```diff
@@ -0,0 +1,116 @@
+            "deprecated": false
+          }
+        },
+        "x_y_z": {
```
"x_y_z_angle"

teoli2003 commented on css/properties/rotate.json:
```diff
@@ -0,0 +1,116 @@
+        },
+        "x_y_z": {
+          "__compat": {
+            "description": "Support for x, y, or z axis name plus angle value",
```
Avoid support (it is implicit): "x, y, or z axis name plus angle value" or "Axis name (x, y, or z) plus angle value"

chrisdavidmills commented:
Oops, sorry for the stupid copy and paste error - rotate is really rotate now ;-)

Fixed the other bits too.
